controllers/propagator/propagation.go

```diff
@@ -69,15 +77,17 @@ func Initialize(kubeconfig *rest.Config, kubeclient *kubernetes.Interface) {
 	attempts = getEnvVarPosInt(attemptsEnvName, attemptsDefault)
 	requeueErrorDelay = getEnvVarPosInt(requeueErrorDelayEnvName, requeueErrorDelayDefault)
 	concurrencyPerPolicy = getEnvVarPosInt(concurrencyPerPolicyEnvName, concurrencyPerPolicyDefault)
+	encryptionEnabled = getEnvVarBool(encryptionEnabledEnvName, encryptionEnabledEnvDefault)
```

Comment:
What was the reason you are making this optional? My preference would be to make this always on and remove the code checking this value.

Reply:
I'll update it. I wasn't sure if it was desirable to be able to shut it off.